product/opni/components/Clusters.vue

```diff
@@ -91,7 +140,7 @@ export default {
       default-sort-by="expirationDate"
       key-field="id"
     />
-    <AddTokenDialog ref="dialog" @save="load" />
+    <EditClusterDialog ref="dialog" @save="load" />
```

Comment:
I used the dialoag for tokens because they were so simple and didn't have much to configure. In rancher for nearly every resource we create and edit resources on a separate page so we're likely going to want to go back to that. We can wait until after we start integrating with rancher.